tests/tests.h

```diff
@@ -187,6 +189,51 @@ namespace Testing
 
 
 
+// Construct a random point, with coordinates between min and max
+template<int dim>
+inline Point<dim> random_point(const double &min=0.0,
+                               const double &max=1.0)
+{
+  Assert(max >= min, ExcMessage("Make sure max>=min"));
+  Point<dim> p;
+  for (unsigned int i=0; i<dim; ++i)
+    p[i] = min+(Testing::rand()*(max-min))/((double)RAND_MAX);
+  return p;
+}
+
+
+
+// Get a positive random value between min and max
+template<type T=double>
+T random_value(const T &min, const T &max)
+{
+  return min+(Testing::rand()*(max-min))/static_cast<T>(RAND_MAX);
```

Comment:
it's a bit easier to read: 

`min+(max-min)*(Testing::rand()/static_cast<T>(RAND_MAX));`